Source/WebCore/css/typedom/transform/CSSMatrixComponent.cpp

```diff
@@ -155,8 +155,7 @@ void CSSMatrixComponent::serialize(StringBuilder& builder) const
 
 ExceptionOr<Ref<DOMMatrix>> CSSMatrixComponent::toMatrix()
 {
-    // FIXME: Implement.
-    return DOMMatrix::fromMatrix(DOMMatrixInit { });
+    return { m_matrix.get() };
```

Comment:
Does this really match https://drafts.css-houdini.org/css-typed-om/#dom-csstransformcomponent-tomatrix ?

The spec looks a bit more complicated than this?

Reply:
I _just_ saw this after landing. I believe some other tests are covering unit checking, will look into this next. Or we can revert this, but I don't think it hurts to keep the incremental change performed here.

Reply:
No need to revert.